.npmignore

```diff
@@ -1,8 +1,8 @@
 .DS_Store
 
 node_modules/
-src/
-test/
+src/**/*.ts
```

Comment:
You might also want to add *.map files to exclude

Reply:
Done!